server/src/infra/repositories/asset.repository.ts

```diff
@@ -174,7 +174,7 @@ export class AssetRepository implements IAssetRepository {
           person: true,
         },
       },
-      withDeleted: !!options.trashedBefore,
+      withDeleted: options.withDeleted ?? !!options.trashedBefore,
```

Comment:
Is this a convenient bug fix?

Reply:
No, just a new feature so that the report takes soft deleted assets into account